src/components/overall-statistics/RaceToMapStat.vue

```diff
@@ -10,8 +10,8 @@
     >
       <template v-slot:body>
         <tbody>
-          <tr v-for="item in stats" :key="item.map">
-            <td>{{ item.map }}</td>
+        <tr v-for="item in stats" :key="item.mapName || item.map">
+          <td>{{ item.mapName || item.map }}</td>
```

Comment:
The indentation change is incorrect, the `<td>` is a sibling of `<player-stats-race-versus-race-on-map-table-cell>`
```suggestion
          <tr v-for="item in stats" :key="item.mapName || item.map">
            <td>{{ item.mapName || item.map }}</td>
```